product_docs/docs/biganimal/release/overview/03a_region_support/index.mdx

```diff
@@ -117,3 +177,22 @@ When using AWS and BigAnimal's cloud account, you can create clusters in the fol
 | Europe (Frankfurt) | eu-central-1 |
 | Europe (Ireland)   | eu-west-1    |
 | Europe (London)    | eu-west-2    |
+
+
+### Google Cloud regions 
```

Comment:
the regions in NA are not correct; see column F here: https://docs.google.com/spreadsheets/d/1FQiRA-OIzX8CWchY1UGxgJvIblUIf5Ie5SSF2FJ_bOE/edit#gid=830197369
currently BAH+GCP supports N Virginia, Belgium, Sydney, Mumbai regions with more coming soon